interface-module/src/main/resources/application.yml

```
@@ -3,7 +3,12 @@ spring:
     name: mini-pay
 
   profiles:
-    active: mysql  # 현재 활성화할 프로파일 지정
+    active: oracle,dev  # 현재 활성화할 프로파일 지정
+
+  #항목	            prod	        dev	            local
```

YAMLlint (1.37.1)

[warning] missing starting space in comment

(comments)